test/viewmodels/v2/IndexPageHelperSpec.scala

```diff
@@ -0,0 +1,1574 @@
+          lateSubmissionPenalty = None,
+          latePaymentPenalty = Some(
+            LatePaymentPenalty(
+              details = Seq.empty
+            )
+          )
+        )
+        val result = pageHelper.getContentBasedOnLatePaymentPenaltiesFromModel(penaltyDetails)(implicitly, vatTraderUser)
+        val parsedHtmlResult = Jsoup.parse(result.body)
+        parsedHtmlResult.select("p.govuk-body").text() shouldBe noActivePaymentPenalty
+      }
+    }
+
+    "display unpaid VAT text and 'how lpp calculated' link" when {
+      val penaltyDetailsUnpaidVAT: PenaltyDetails = PenaltyDetails(
+        totalisations = None,
+        lateSubmissionPenalty = None,
+        latePaymentPenalty = Some(
+          LatePaymentPenalty(
+            details = Seq(
+              LPPDetails(
+                principalChargeReference = "12345678",
+                penaltyCategory = LPPPenaltyCategoryEnum.LPP1,
+                penaltyChargeCreationDate = LocalDate.of(2022, 1, 1),
+                penaltyStatus = LPPPenaltyStatusEnum.Posted,
+                penaltyAmountPaid = Some(0),
+                penaltyAmountOutstanding = Some(144.21),
+                LPP1LRDays = None,
+                LPP1HRDays = None,
+                LPP2Days = None,
+                LPP1LRCalculationAmount = None,
+                LPP1HRCalculationAmount = None,
+                LPP1LRPercentage = None,
+                LPP1HRPercentage = None,
+                LPP2Percentage = None,
+                communicationsDate = LocalDate.of(2022, 1, 1),
+                penaltyChargeDueDate = LocalDate.of(2022, 1, 1),
+                appealInformation = None,
+                principalChargeBillingFrom = LocalDate.of(2022, 1, 1),
+                principalChargeBillingTo = LocalDate.of(2022, 1, 1),
+                principalChargeDueDate = LocalDate.of(2022, 1, 1)
+              )
+            )
+          )
+        )
+      )
+      "user has outstanding vat to pay" in {
+        val result = pageHelper.getContentBasedOnLatePaymentPenaltiesFromModel(penaltyDetailsUnpaidVAT)(implicitly, vatTraderUser)
+        val parsedHtmlResult = Jsoup.parse(result.body)
+        parsedHtmlResult.select("p.govuk-body").get(0).text shouldBe unpaidVATText
+        parsedHtmlResult.select("a.govuk-link").text shouldBe howLppCalculatedLinkText
+        //TODO: change this when we have link to calculation page
+        parsedHtmlResult.select("a.govuk-link").attr("href") shouldBe "#"
+      }
+
+      "client has outstanding vat to pay" in {
+        val result = pageHelper.getContentBasedOnLatePaymentPenaltiesFromModel(penaltyDetailsUnpaidVAT)(implicitly, agentUser)
+        val parsedHtmlResult = Jsoup.parse(result.body)
+        parsedHtmlResult.select("p.govuk-body").get(0).text shouldBe agentClientUnpaidVATText
+        parsedHtmlResult.select("a.govuk-link").text shouldBe howLppCalculatedLinkText
+        //TODO: change this when we have link to calculation page
```

Comment:
Same as above

Reply:
Its not to do with the calculation page, it's external guidance. I'll update the comment so it's clearer